src/token/api/src/account.rs

```diff
@@ -27,7 +30,16 @@ impl From<Principal> for Account {
 
 impl Display for Account {
     fn fmt(&self, f: &mut Formatter) -> std::fmt::Result {
-        write!(f, "{}", self.owner)
+        if self.subaccount == DEFAULT_SUBACCOUNT {
+            write!(f, "Account({})", self.owner)
+        } else {
+            write!(f, "Account({}, ", self.owner)?;
+            for b in self.subaccount {
+                write!(f, "{b:02X}")?;
+            }
```

Comment:
If we write the comma before the sub account it won't look so naff if there is no sub accounts
```suggestion
            write!(f, "Account({}", self.owner)?;
            for b in self.subaccount {
                write!(f, ", {b:02X}")?;
            }
```

Reply:
If there's no subaccount, another brunch is executed. Adding `,` before each byte in subaccount wouldn't look nice....